src/spec/product-type-importer.spec.coffee

```diff
@@ -0,0 +1,113 @@
+Promise = require 'bluebird'
+ProductTypeImporter = require '../lib/product-type-import'
+{SphereClient} = require 'sphere-node-sdk'
+
+errMissingCredentials = 'Missing configuration in env variables'
+
+argv =
+  clientId: process.env.SPHERE_CLIENT_ID
+  clientSecret: process.env.SPHERE_CLIENT_SECRET
+  projectKey: process.env.SPHERE_PROJECT_KEY
```

Comment:
@emmenko we used to do that but it is not very useful if you run it for example in iron.io. as you can set env vars but can not generate files on file system before the job runs. I would rather start using env vars.